apps/web/app/(app)/environments/[environmentId]/(actionsAndAttributes)/actions/components/ActionClassesTable.tsx

```diff
@@ -45,16 +45,21 @@ export default function ActionClassesTable({
 
   return (
     <>
-      <LoadingWrapper isLoading={isLoading} error={error}>
+      <LoadingWrapper error={error}>
         {!isViewer && (
           <div className="mb-6 text-right">
             <Button
               variant="darkCTA"
+              className={
+                isLoading
```

Comment:
1. Matti said we should always use cn instead of plain JS.
2. Our button comes with a prebuilt loading state. You could just pass `loading={isLoading}` to it and it will handle the styling. Once again, this keeps the UI/UX consistent throughout the app.

Reply:
changed this to use the loading prop